src/snowcli/cli/snowpark/procedure_coverage/clear.py

```diff
@@ -5,17 +5,12 @@
 
 from snowcli import config, utils
 from snowcli.config import AppConfig
-from snowcli.utils import conf_callback, generate_deploy_stage_name
+from snowcli.utils import check_for_connection, generate_deploy_stage_name
 from snowcli.output.printing import print_db_cursor
 
 from . import app
+from ...sql import ConnectionOption
```

Comment:
It doesn't look like the best way for this import.

Reply:
Good catch